chainer_chemistry/models/ggnn.py

```diff
@@ -28,26 +28,27 @@ class GGNN(chainer.Chain):
     """
     NUM_EDGE_TYPE = 4
 
-    def __init__(self, out_dim, hidden_dim=16,
-                 n_layers=4, n_atom_types=MAX_ATOMIC_NUM, concat_hidden=False,
-                 weight_tying=True):
+    def __init__(self, out_dim, hidden_dim=16, n_layers=4,
+                 n_atom_types=MAX_ATOMIC_NUM, activation=functions.identity,
+                 concat_hidden=False, weight_tying=True):
```

Comment:
sorry, could you add `activation` as final argument to keep backward compatibility as much as possible?